biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -145,6 +146,21 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	/**
+	 * @return the java release target corresponding to this EE
+	 */
+	public OptionalInt getReleaseTarget() {
+		Version version = getCapabilityVersion();
+		int major = version.getMajor();
+		if (major > 8) {
```

Comment:
```suggestion
		if (major > 1) {
```

I think testing for major > 1 is more general.